inprocess/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -697,6 +698,20 @@ public StatsTraceContext statsTraceContext() {
       public int streamId() {
         return -1;
       }
+
+      /**
+       * A hint to the stream that specifies how many bytes must be queued before
+       * {@link StreamListener#onReady()} will be called. A stream may ignore this property if
+       * unsupported. This must be set before any messages are sent.
+       *
+       * @param numBytes The number of bytes that must be queued. Must be a
+       *                 positive integer.
+       */
+      @Override
+      @ExperimentalApi("https://github.com/grpc/grpc-java/issues/11021")
```

Comment:
This method isn't visible to users (the class is not public), so no need to mark it `@ExperimentalApi`.